src/main/target/OMNIBUS/target.h

```diff
@@ -169,9 +169,10 @@
 
 //#define AVOID_UART3_FOR_PWM_PPM // Disable this for using UART3
 
-#define SPEKTRUM_BIND_PIN       UART3_RX_PIN
-
-#define BINDPLUG_PIN            PB0
+#define USE_SPEKTRUM_BIND
```

Comment:
I think with this implementation, `USE_SPEKTRUM_BIND` should be enabled for all targets that have enough flash space and support Spektrum RX.